projects/azure/base-images/test-base/Dockerfile

```diff
@@ -0,0 +1,72 @@
+# This is development dockerfile
+# original copy is kept inside build-config
```

Comment:
Is this comment still true?

Reply:
yes, I just copy paste it each time and got bored to do so. So added it so it would need.